packages/web-components/src/slider/slider.spec.md

```diff
@@ -0,0 +1,118 @@
+# Slider
+
+## Component Description
+
+The Slider component allows users to quickly select a value by dragging an icon across a bar. It is often used when setting values with a relaxed precision such as audio volume and screen brightness.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/IBCBJxEbPKS7CvLHG55Wn9/Slider?node-id=513%3A510&t=hCOhOiQJVCVNVVJP-0)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr size: "small" "medium" | "medium"
+- @attr vertical: boolean | false
+- @attr current-value: number
+- @attr min: number
+- @attr max: number
+- @attr step: number | 1
+  - If set, slider appearance has ticks for each step value
+- @attr disabled: boolean | false
+- @attr mode: "single-value" "range" | "single-value"
+
+### Outputs
+
+None
+
+### Events
+
+- @change: Fires whenever the slider's value is changed
+
+### Slots
+
+- default: the primary slot intended for the `input` element
```

Comment:
Are we forking the FAST template and slotting elements as a pattern for how form elements work? This feels more like a construct from the react implementation where what is being returned is an input...but you can "slot" your own if you want. This won't work the same w/ custom elements and this is not how the FAST Form Associated elements work or will work.

Reply:
Gotcha, makes sense. Thanks.

Reply:
Updated it to reflect that the default slot reflects child elements of the component, such as the SliderLabel component.